website/css/style.css

```diff
@@ -0,0 +1,563 @@
+
+/**
+ * Global styles.
+ */
+html,body {
+    background-color: #FAFAFA;
+    height: 100%;
+}
+
+a, a:link, a:visited, a:hover, a:active {
+    color: inherit;
+    text-decoration: none;
+}
+
+h {
+    font-weight: inherit;
+}
+
+pre {
+    overflow: auto;
+}
+
+/**
+ * Main content.
+ */
+.psr-main {
+    display: flex;
+    /* Take up at least one screen (minus header) so the vert line separating
+       left nav from content doesn't stop off when the left-nav is collapsed. */
+    min-height: calc(100% - 68px);
+}
+
+/**
+ * #header
+ * Top header.
+ */
+#header {
+    /* Theme primary text inverse - 100% white */
+    color: #FFF;
+    /* Theme primary - Google Blue 600 */
+    background-color: #3B78E7;
+ 
+    height: 68px;
+    /* Same as #content-container, otherwise bugs on h-scroll. */
+    min-width: 800px;
+    width: 100%;
+}
+
+/* Hack to enable vertical alignment. This and the `#header *` selector are
+   crazy-fragile. You have been warned. */
+#header:before {
+  content: "";
+  display: inline-block;
+  vertical-align: middle;
+  height: 100%;
+}
+
+#header * {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+#header .psr-headerIcon {
+    margin-left: 24px;
+    margin-right: 24px;
+
+    background-image: url("/google-cloud-powershell/img/gcp-logo.png");
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 25px;  min-width: 25px;
+    height: 25px; min-height: 25px;
+}
+
+#header .psr-headerText {
+    margin-left: 24px;
+
+    font-size: 20px;
+    line-height: 36px;
+    font-family: "Open Sans";
+    font-weight: 100;
+}
+
+/**
+ * #left-nav, #left-nav-top, #left-nav-bottom
+ * The left nav pane, browsing cmdlets. Broken into two sections.
+ */
+#left-nav {
+    border-bottom: 1px solid rgba(0,0,0,0.08);
```

Comment:
It's not a super nit. The preferred way would be the RGB hex, as that is the most concise. 

However, I need to use `rgba` in order to specify an opacity. The mocks use terms like "black, 8%". And converting that to hex is hard.